stages/07.extra-scripts/examples/extra-script-example.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# kuiper2.0 - Embedded Linux for Analog Devices Products
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+# Author: Larisa Radu <larisa.radu@analog.com>
+
+
+# This script is run inside 'chroot'. 
+# - It works as if Kuiper is running on a system with a few limitations. 
+# - Check 'chroot' documentation for more informations.
+# This script is run as root, there is no need to use 'sudo' command.
+# Current directory is '/' (root) of the Kuiper rootfs.
+# If a file needs to be copied, it should be placed inside 'adi-kuiper-gen'.
+# If a variable from the configuration file is needed, it should be passed as a parameter at script calling.
+# - modify stages/07.extra-scripts/run.sh to pass parameters (see example)
+# - assign parameter value to a variable name
+# At this stage the Kuiper image is not yet partitioned. In order to modify what will be in the boot partition access /boot folder.
+# This script will not be in the resulted image. If this is necessary it should be copied manually.
+
+# Assign script parameter value to a variable (as passed in the example in stages/07.extra-scripts/run.sh)
+# TARGET_ARCHITECTURE=$1
```

Comment:
Since all the variables used are exported in [kuiper-stages.sh](https://github.com/analogdevicesinc/adi-kuiper-gen/pull/145/commits/e22b565a61134961c1ca2b6e4da04e22cf557e0d#diff-9a885fac99e89a5fdb5e0256bfec1d5416de69c2ac4f6d0746bd5c50f1fd17ab), I was thinking is enough to add a link to that list (kuiper-stages.sh from line 16) in the example file, so people will have a list with all already initialized variables that can be used in the extra script.

Reply:
Great idea, I will exemplify how to source the config file directly in the example script.